app/models/inventory-batch.js

```diff
@@ -16,7 +16,7 @@ export default AbstractModel.extend({
     },
     inventoryItemTypeAhead: {
       presence: {
-        if(object) {
+        unless(object) {
```

Comment:
It is very possible that I am missing something, but my understanding is that `unless` is the inverse of `if`. Meaning that `unless` will skip validation if the method returns `true` as opposed to `if` which will skip validation if the method return `false`. The pseudo code interpretation would be "Unless the object has invoice items, trigger validation".

An example of this is
https://github.com/mkly/hospitalrun-frontend/blob/af38a215463682ef4f31b6774078cb3145ae4497/tests/unit/models/inventory-batch-test.js#L34-L36
The validation is skipped because `invoiceItems` is non empty and therefore `haveInvoiceItems` returns `true`
The opposite is
https://github.com/mkly/hospitalrun-frontend/blob/af38a215463682ef4f31b6774078cb3145ae4497/tests/unit/models/inventory-batch-test.js#L32
The validation here is run because `invoiceItems` is not set(empty) and therefore `haveInvoiceItems` returns `true`. This is why the assertion of `testInvalidPropertyValues` passes here but not in the above linked test. Because in this test the validation did run.

I probably could have made this pull request easier to follow by just appending `!` in order to invert the call to `haveInvoiceItems`. I just thought using `unless` was a little clearer to someone coming into this fresh. That said, `unless` is not a conditional keyword in JavaScript like it is in ruby and perl. Let me know if I'm way off here and my apologies in advance if I am missing the obvious. And if you would prefer I can just invert with `!` and use `if` instead of `unless`.

Thanks again for taking the time to look this over.